### PR TITLE
US-24-create-method-to-get-birthday-by-duration-cli

### PR DESCRIPTION
**Pull request by TaniaSakun**
## Description:
1. Added a method to show the birthdays by day
2. Added small validation, that by default gets the birthday for 7 days duration

### CODE REVIEW

File: apps/cli/app/services.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 from beaupy import ValidationError, select
-from core import Actions, ContactPayload, ResponseType, controller
+from core import Actions, AllBirthdaysPayload, ContactPayload, ResponseType, controller
+from prettytable import PrettyTable
```
Comment (iMykhailychenko) [inline apps/cli/app/services.py:3]: you should also run `pip freeze > requirements.txt` to add this dependancy

File: apps/cli/app/services.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 from beaupy import ValidationError, select
-from core import Actions, ContactPayload, ResponseType, controller
+from core import Actions, AllBirthdaysPayload, ContactPayload, ResponseType, controller
```
Comment (iMykhailychenko) [inline apps/cli/app/services.py:2]: This import will be updated, unfortunately

File: apps/cli/app/services.py (hunk 1 above)
Comment (TaniaSakun) [reply apps/cli/app/services.py:4]: Done

File: apps/cli/app/services.py (hunk 2 above)
Comment (TaniaSakun) [reply apps/cli/app/services.py]: Done

**Review by iMykhailychenko — APPROVED** [review]

**Pull request merged**